infrastructure/aat.tfvars

```diff
@@ -12,4 +12,6 @@ idam_redirect_url = "https://evidence-sharing-preprod.sscs.reform.hmcts.net"
 hours_start_time = "0"
 hours_end_time = "23"
 
-bundled_letters_on = "true"
\ No newline at end of file
+bundled_letters_on = "true"
+
+letters_on = "true"
```

Comment:
the variable is not defined. You need to replicate the same done for the `BUNDLED_LETTERS_ON`